### PR TITLE
Collapse/Expand all diff blocks in one click related to issue #1

### PR DESCRIPTION
**Pull request by xserrat**
Hi @picheli20 ,

I've added another button `Collapse/Expand All` in order to `collapse/expand` all diff blocks in one click related to the issue #1 .

Thanks!

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -46,17 +48,29 @@ function initialSetup() {
   $('#closeAll').click(() => {
       $('.gct-folder').removeClass('gct-folder-open');
   });
+
```
Comment (picheli20) [inline app/app.js:51]: Could you add this function here? So in this way we open automatically when the file is clicked:

```js
  $('.gct-file-name').click(obj => {
    var href = $(obj.currentTarget)[0].getAttribute("href");
    var file = $(`.file-info > a[href="${href}"]`).parent().parent().parent();
    if ($(file).hasClass('open Details--on')) {
      $(file).removeClass('open Details--on');
    }
  });

```

File: app/app.js [hunk 2]
```diff
@@ -139,3 +153,39 @@ function mergeObjects(og, so) {
     }
     return og;
 }
+
+function areDiffBlocksCollapsed()
```
Comment (picheli20) [inline app/app.js:157]: Could add the `{` at the same line (`function areDiffBlocksCollapsed() {`), just to keep the code consistency :)

**Review by picheli20 — CHANGES_REQUESTED** [review]
Great fix, just a quickly improvement then we can merge it

File: app/app.js (hunk 1 above)
Comment (xserrat) [reply app/app.js:51]: That's a great idea, more useful. 👍

File: app/app.js (hunk 2 above)
Comment (xserrat) [reply app/app.js:157]: Totally.

**Comment by picheli20** [comment]
Fixed on https://github.com/picheli20/github-code-tree/commit/cc6ecbda7520ca9651f2b50e53e83068ebbbb4f1

**Pull request closed without merging**